### ISSUE TITLE
Unable to train the network with the pre-trained weights except for darknet19_448

### ISSUE THREAD

**Issue by prateekgupta891** [pjreddie/darknet#1678]
I was trying to train a modified yolo network with some pre-trained weights for the given network only. But it's doesn't train and directly saves the weights.
![image](https://user-images.githubusercontent.com/32310164/59909084-84dc1b00-942c-11e9-8fd4-4ccff37310d6.png)

But if i give darknet19-448.conv.23 as the training weights (which surely are for different network configuration), it starts training.
![image](https://user-images.githubusercontent.com/32310164/59909117-9291a080-942c-11e9-8b3d-b1708b07f310.png)

It even trains if i try grounds up training, without giving any pre-trained weights.
![image](https://user-images.githubusercontent.com/32310164/59909333-19467d80-942d-11e9-8533-8ee9e84736c7.png)

Still if i try some other pre-trained weights like (Alexnet and Yolo) it doesn't train and directly saves the weights.
![image](https://user-images.githubusercontent.com/32310164/59909156-a6d59d80-942c-11e9-9cd6-3f5dc6d19198.png)
![image](https://user-images.githubusercontent.com/32310164/59909354-25323f80-942d-11e9-9e7c-9bc8b1fe7e06.png)

I hope somebody can tell me what am i doing wrong?
Because the pre-trained weights for my custom network works well enough to detect the objects.

**Comment by AlexeyAB** [comment]
You reached maximum iterations, so training is stopped immediately..

Increase `max_batches=` in cfg-file
or use flag ` -clear`

**Comment by joeljosephjin** [comment]
I have same problem.  upon typing -
./darknet detector train data/obj.data yolov2-obj.cfg yolov2.weights


Output -   30 conv     95       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x  95 0.033 BF
  31 detection
mask_scale: Using default '1.000000'
Total BFLOPS 28.246 
Loading weights from yolov2.weights...
 seen 32, trained: 32013 K-images (500 Kilo-batches_64) 
Done! Loaded 32 layers from weights-file 
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
Saving weights to backup//yolov2-obj_final.weights

**Comment by joeljosephjin** [comment]
And when I gave the flag -clear the process got killed.

  30 conv     95       1 x 1/ 1     13 x  13 x1024 ->   13 x  13 x  95 0.033 BF
  31 detection
mask_scale: Using default '1.000000'
Total BFLOPS 28.246 
Loading weights from yolov2.weights...
 seen 32, trained: 32013 K-images (500 Kilo-batches_64) 
Done! Loaded 32 layers from weights-file 
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
Resizing, random_coef = 1.40 
608 x 608 
Loaded: 1.464145 seconds
Killed
